dataactcore/models/jobTrackerInterface.py

```diff
@@ -275,3 +275,6 @@ def getSubmissionStatus(self,submissionId):
         if statuses["finished"] == len(jobIds)-skip_count: # need to account for the jobs that were skipped above
             return "finished"
         return "unknown"
+
+    def getSubmissionsByUserId(self, user_id):
+        return self.session.query(Submission).filter(Submission.user_id == user_id).all()
```

Comment:
Either way, as long as we don't leave two copies and the copy we keep gets a docstring I'm happy.